cli/tree/src/main/scala/edu/rpi/tw/twxplore/cli/tree/command/EtlCommand.scala

```diff
@@ -15,8 +23,230 @@ object EtlCommand extends Command {
 
   val args = new Args()
 
+  def replaceComma(str: String, startIndex: Int, endIndex: Int): String = {
+    val result: String = str.substring(0, startIndex) + str.substring(startIndex+1, endIndex).replace(",", "+") + str.substring(endIndex+1)
+    result
+  }
+
   def apply(): Unit = {
-//    val pipelineName = args.pipelineName.toLowerCase()
+    val pipelineName = args.pipelineName.toLowerCase()
+    val dataDirectoryPath = Paths.get(args.dataDirectoryPath)
+    val source = scala.io.Source.fromFile(args.dataDirectoryPath)
+    var treeList: ListBuffer[Tree] = new ListBuffer[Tree]()
+    var treeSpeciesList: mutable.HashMap[String, TreeSpecies] = new mutable.HashMap()
```

Comment:
Not a list..

Reply:
Are you referring to the misnaming of treeSpeciesList, or the usage of the mutable ListBuffer to store trees?

Reply:
treeSpeciesList is a map, yes.